subprojects/snapshots/src/test/groovy/org/gradle/internal/snapshot/RegularFileSnapshotTest.groovy

```diff
@@ -16,11 +16,12 @@
 
 package org.gradle.internal.snapshot
 
+import org.gradle.internal.file.FileMetadataSnapshot.AccessType
 import org.gradle.internal.hash.HashCode
 
 class RegularFileSnapshotTest extends AbstractCompleteSnapshotWithoutChildrenTest {
     @Override
     protected CompleteFileSystemLocationSnapshot createInitialRootNode(String absolutePath) {
-        return new RegularFileSnapshot(absolutePath, PathUtil.getFileName(absolutePath), HashCode.fromInt(1235), new FileMetadata(1, 2))
+        return new RegularFileSnapshot(absolutePath, PathUtil.getFileName(absolutePath), HashCode.fromInt(1235), new FileMetadata(1, 2), AccessType.DIRECT)
```

Comment:
Shall we add tests for symlinked content?

Reply:
The `accessType` does not change the behaviour here, so I would say we don't need to add those tests.